src/axolotl/monkeypatch/models/qwen3_5/modeling.py

```
@@ -0,0 +1,468 @@
+
+    def patched_forward(
+        self,
+        hidden_states: torch.Tensor,
+        cache_params=None,
+        cache_position: Optional[torch.LongTensor] = None,
+        attention_mask: Optional[torch.Tensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
+    ):
+        hidden_states = apply_mask_fn(hidden_states, attention_mask)
+
+        batch_size, seq_len, _ = hidden_states.shape
```

Ruff (0.15.2)

[warning] Unpacked variable `batch_size` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)